### ISSUE TITLE
It doesn't work in some web sites.

### ISSUE THREAD

**Issue by skanskan** [rogershen/chrome-regex-search#8]
Hello.
It seems it doesn't work in this simple web:
https://cran.r-project.org/
It just doesn't find anything.

**Comment by dopeghoti** [comment]
Unable to replicate.  Going to that site, searches work as expected.

**Comment by dennisrichter** [comment]
I also cannot replicate the problem on the before mentioned page https://github.com/rogershen/chrome-regex-search/

But i can on this site: http://www.antiquariat-schwarz.de/
May be because of the medieval frameset or table layouts, not sure here.

**Comment by Takophiliac** [comment]
Here's another sample:
Go to http://www.kingswaysoft.com/blog
Nothing in the bottom iframe can be searched. (search for "comments" for example)

**Comment by cxw42** [comment]
@Takophiliac That is at least partly because the [`manifest.json`](https://github.com/rogershen/chrome-regex-search/blob/master/src/manifest.json#L40) doesn't specify the `all_frames` attribute in `content_scripts` (see [the docs](https://developer.chrome.com/extensions/content_scripts), and search for `all_frames`).  Although I doubt that's a one-line fix :) .

**Comment by huhx** [comment]
yes, java docs website also arise the same problem. https://docs.oracle.com/javase/7/docs/api/

**Comment by micsthepick** [comment]
https://www.cs.cmu.edu/~rbd/doc/nyquist/ also does not work, probably because of the "medieval frameset".

**Comment by SGM3** [comment]
https://docs.liferay.com/portal/7.2-latest/definitions/

Is another example. Chrome's default search works fine, but the search seems to get stuck on the first iframe?
